stack.go

```diff
@@ -0,0 +1,30 @@
+package slogx
+
+import (
+	"fmt"
+	"log/slog"
+	"runtime"
+	"strings"
+)
+
+const (
+	KeyStack = "stack"
+	trim     = 3
+)
+
+// Stack returns a stack trace formatted as panic output.
+// It excludes a call of slogx.Stack() itself.
+func Stack() slog.Attr {
+	buf := make([]byte, 2048)
```

Comment:
Why 2048?
Looking at examples in stdlib I see these cases:
- start with `1024` and then grow if not enough in [runtime/debug](https://cs.opensource.google/go/go/+/refs/tags/go1.20.6:src/runtime/debug/stack.go;l=22) and [os/exec](https://cs.opensource.google/go/go/+/refs/tags/go1.20.6:src/os/exec/exec.go;l=386)
- `64 << 10` (65536) in [net/http](https://cs.opensource.google/go/go/+/refs/tags/go1.20.6:src/net/http/server.go;l=1852)
- `2 << 10` (2048) in [database/sql](https://cs.opensource.google/go/go/+/refs/tags/go1.20.6:src/database/sql/sql.go;l=3397)

To me it looks like we should either do same as net/http (use 64KB) or os/exec (grow as needed).